docs/platform/howto/list-monitoring.md

```diff
@@ -67,7 +67,7 @@ You can export logs and metrics **to an Aiven service**:
 - Send logs to [Aiven for M3](/docs/products/m3db).
 - Visualize logs with [Aiven for Grafana](/docs/products/grafana).
 
-You can also export the data **to external tools**, see [metrics integrations](/docs/platform/howto/metrics-integrations).
+You can also export the data **to external tools**, see [metrics integrations](/docs/platform/howto/list-monitoring).
```

Comment:
Nitpicking: Bold text is used for GUI/console labels and seems using it for emphasizing plain content might be misleading for the readers. I would reserve it for GUI items so that the users can form proper associations.
Also, the link is not actually taking the users directly to any info on exporting data to external tools. Perhaps we could replace it with a list of links to articles that actually cover the topic?